dashboard/cron.py

```diff
@@ -417,6 +417,15 @@ def update_resource_access(self):
             logger.debug(f'resource_access_df:\n'
                          f'{resource_access_df}\n'
                          f'{resource_access_df.dtypes}')
+            
+            # only keep access events generated by students
+            student_enrollment_df = pd.read_sql(
+                'select user_id, course_id '
+                f'from user where enrollment_type=\'StudentEnrollment\'',
```

Comment:
Optional: `StudentEnrollment` could be constant rather than buried in the code

Reply:
Done with [b492ae3](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1395/commits/b492ae39078458c83f7193f051141e17ddb98c36)